src/main/java/seedu/address/logic/commands/StartCommand.java

```diff
@@ -27,55 +24,59 @@
             + "Parameters: DECK NAME. If no argument is supplied, a random deck will be chosen.\n"
             + "Example: " + COMMAND_WORD + " physics";
 
+    public static final String MESSAGE_NO_FLASHCARDS = "No flashcards to test!";
+
     public static final String MESSAGE_START_TEST_SUCCESS = "Starting test...";
 
     private final AddressBookParser addressBookParser;
 
-    private final String deckName; // TODO: will integrate after deck class is completed
+    private final String tagName;
 
     public StartCommand(AddressBookParser addressBookParser) {
-        this.deckName = "Test"; // TODO: will get a random deck name from list of decks
+        this.tagName = "";
```

Comment:
Try not to initialize variables to dummy values if possible.

Reply:
Equivalent to `this.tagName = new String();`. Omitting the line will require the variable to be non-final and additional methods to check for null variables would be required.